Scripts/Runtime/Entities/TaskSystem/AbstractTaskDriver.cs

```diff
@@ -48,35 +54,70 @@ public abstract class AbstractTaskDriver : AbstractAnvilBase
         /// </summary>
         public AbstractTaskSystem TaskSystem { get; }
         
+        /// <summary>
+        /// Reference to the associated <see cref="World"/>
+        /// </summary>
+        public World World { get; }
+
         internal CancelRequestsDataStream CancelRequestsDataStream { get; }
+        internal List<AbstractTaskStream> TaskStreams { get; }
+        internal TaskDriverCancellationPropagator CancellationPropagator { get; private set; }
 
         protected AbstractTaskDriver(World world, AbstractTaskSystem abstractTaskSystem)
         {
-            TaskSystem = abstractTaskSystem;
+            World = world;
             
+            //TODO: #112 (anvil-csharp-core) Extract to Anvil-CSharp Util method -Used in AbstractJobConfig as well
+            m_TypeString = GetType().Name;
+            TaskSystem = abstractTaskSystem;
             Context = TaskSystem.RegisterTaskDriver(this);
             
-            //TODO: #71 - Let the TaskFlowGraph create this for us.
             m_SubTaskDrivers = new List<AbstractTaskDriver>();
+            TaskStreams = new List<AbstractTaskStream>();
+            m_JobConfigs = new List<AbstractJobConfig>();
+            
+            TaskStreamFactory.CreateTaskStreams(this, TaskStreams);
             CancelRequestsDataStream = new CancelRequestsDataStream();
             
             m_TaskFlowGraph = world.GetOrCreateSystem<TaskFlowSystem>().TaskFlowGraph;
-            m_TaskFlowGraph.CreateTaskStreams(TaskSystem, this);
-            m_TaskFlowGraph.RegisterCancelRequestsDataStream(CancelRequestsDataStream, TaskSystem, this);
+            m_TaskFlowGraph.RegisterTaskDriver(this);
```

Comment:
Since this seems to be the only place that you're using the graph should the system just register the task driver with the graph when it gets registered with the system?
Then you don't need reference to the graph.
(Ex: in `TaskSystem.RegisterTaskDriver`)

Reply:
Possibly. 

Added TODO to investigate for #66, #67 and/or #68 and linked to this comment.